.github/ISSUE_TEMPLATE/security-issue.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Security Issue
+about: If you think you found a security vulnerability with Metabase
+title: ''
+labels: ''
```

Comment:
Can we add `Security` as label, just in case someone opens an issue, then we can filter on it - and easier to see?